drivers/audio/Kconfig

```diff
@@ -149,7 +149,7 @@ config CS43L22_NUM_BUFFERS
 
 config CS43L22_WORKER_STACKSIZE
 	int "CS43L22 worker thread stack size"
-	default 768
```

Comment:
It is inappropriate to increase the size of this tuned stack size.  Let's not do that.